types/react-navigation/index.d.ts

```diff
@@ -262,7 +263,7 @@ export type NavigationStackScreenOptions = NavigationScreenOptions & {
   headerTitleStyle?: Style,
   headerTintColor?: string,
   headerLeft?: React.ReactElement<any>,
-  headerBackTitle?: string,
+  headerBackTitle?: string | null,
```

Comment:
It is described in the [document](https://reactnavigation.org/docs/navigators/stack#headerBackTitle) and it does work. [Maybe types](https://flow.org/en/docs/types/maybe/) accept the provided type as well as null or undefined. Typescript do not accept null unless specified.